userApp/imports/api/InternalNotification/publish.js

```diff
@@ -0,0 +1,29 @@
+import { Tables } from '/imports/api/Tables';
+import { NOTIFICATION_TYPE__TABLE_INVITATION } from './'
+
+Meteor.publish('internalNotifications', function (roomId) {
+  //NOTE : we should consider using events emmiter for consitency here instead of 
+  //collection observers 
+
+  //FOR now only new tables invitation 
+  let initializing = true;
+  const handle = Tables.find().observeChanges({
```

Comment:
Is this synchronous?

Reply:
no its not why ?

Reply:
the purpose of initializing flag is not to sent invitation for existing tables, just for new one